README.md

```diff
@@ -7,20 +7,70 @@
 Free SVG icon font for popular brands. See them all on one page at <a href="https://simpleicons.org">SimpleIcons.org</a>. Contributions, corrections & requests can be made on GitHub. Started by <a href="https://twitter.com/bathtype">Dan Leech</a>.</p>
 </p>
 
-## Usage
+## Setup
+
+### From CDN
+
+```html
+<html>
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet"
+          href="https://cdn.jsdelivr.net/npm/simple-icons-font/font/simple-icons.min.css"
+          type="text/css">
+  </head>
+  <body>
+    <i class="simpleicons simpleicons-simpleicons simpleicons--color"></i>
+  </body>
+</html>
+```
 
-### General Usage
+### Manual download
+
+1. Download the latest version of the package from
+[here][npm-registry-tarball-link], extract it and move the content of the
+`font` folder to your desired location. 
+1. Reference the CSS file in a `link` tag of your HTML:
+
+```html
+<html>
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="simple-icons.min.css" type="text/css">
+  </head>
+  <body>
+    <i class="simpleicons simpleicons-simpleicons simpleicons--color"></i>
+  </body>
+</html>
```

Comment:
I would reduce this down to just the `<i>` tag and move it to the "Usage" section, as this is how you use it regardless of how you set it up.

```suggestion
    <i class="simpleicons simpleicons-simpleicons simpleicons--color"></i>
</html>
```